pkg/innerring/processors/neofs/process_assets.go

```diff
@@ -86,7 +86,6 @@ func (np *Processor) processWithdraw(withdraw *neofsEvent.Withdraw) {
 	}
 
 	// create lock account
-	// fixme: check collision there, consider reversed script hash
```

Comment:
I still afraid of the collisions between lock accounts and actual user accounts, but have no good solution for that.